Lang/es/ticket.php

```diff
@@ -573,4 +571,21 @@
     "reject_duplicate_emails_desc" => "Impedir que los correos electrónicos se importen más de una vez, detectado en función ID del mensaje.",
     "duplicate_email_detected"  => "Correo duplicado que ya se ha importado.",
 
+    /*
+     * 5.5.0
+     */
+    "rate_limiting"             => "Rate Limiting",
+    "max_requests"              => "Max Requests",
+    "max_requests_desc"         => "Set the maximum number of requests before a user is throttled.",
+    "decay_time"                => "Decay Time",
+    "decay_time_desc"           => "The number of minutes until the available attempts are reset.",
+    "exclude_addresses"         => "Exclude Addresses",
+    "exclude_addresses_desc"    => "A list of email addresses which are excluded from rate limits.",
+    "enable_throttling"         => "Activar limitación",
+    "enable_throttling_api_desc" => "Limita el número de mensajes que un usuario puede publicar en un periodo de tiempo determinado. Consulta <a href='https://docs.supportpal.com/current/App+Configuration#Throttling' target='_blank'>la configuración de la limitación</a> para más información sobre cómo ajustar los límites.",
+    "close_without_notify"      => "Close (without notification)",
```

Comment:
Cerrar (sin enviar notificación)

Reply:
This might be better as `Cerrar (sin notificación)` to avoid it being too long.